static/usage/v7/item/content-types/supporting-visuals/demo.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Item</title>
+    <link rel="stylesheet" href="../../../../common.css" />
+    <script src="../../../../common.js"></script>
+    <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@7/dist/ionic/ionic.esm.js"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@7/css/ionic.bundle.css" />
+  </head>
+
+  <body>
+    <ion-app>
+      <ion-content>
+        <div class="container">
+          <ion-list style="width: 40%">
+            <ion-item>
+              <ion-icon aria-hidden="true" name="airplane" slot="start"></ion-icon>
+              <ion-label>Airplane Mode</ion-label>
+            </ion-item>
+            <ion-item>
+              <ion-icon aria-hidden="true" name="wifi" slot="start"></ion-icon>
+              <ion-label>Wi-Fi</ion-label>
+            </ion-item>
+            <ion-item>
+              <ion-icon aria-hidden="true" name="bluetooth" slot="start"></ion-icon>
+              <ion-label>Bluetooth</ion-label>
+            </ion-item>
+            <ion-item>
+              <ion-icon aria-hidden="true" name="call" slot="start"></ion-icon>
+              <ion-label>Cellular</ion-label>
+            </ion-item>
+          </ion-list>
+
+          <ion-list style="width: 40%">
+            <ion-item>
+              <ion-avatar aria-hidden="true" slot="start">
+                <img alt="" src="https://ionicframework.com/docs/img/demos/avatar.svg" />
```

Comment:
Why add an empty `alt` property?

Reply:
Thread sniping, but isn't the `img` already hidden from screen readers by the `aria-hidden="true"` on the parent element?

Reply:
Ahh I see because otherwise it fails a11y for having no alt tag. Makes sense.